src/gameplay/game.ts

```diff
@@ -2283,8 +2300,73 @@ class Game extends Room {
     }
     return newRating;
   }
+
+  getPlayersOnMission(missionNum: number): Set<string> {      
+    const set = new Set<string>();
+    const VH = this.voteHistory; //brevity
+
+    if(VH === undefined) {
+      return set; // empty set.
+    }
+  
+    for (const player in VH) {
+      //TODO: write check to ensure player really is in this.PlayersInGame.
```

Comment:
It's ok, we can assume that the players in VH are actual players in the game. 

Can remove this comment.